backend/infrahub/events/node_action.py

```diff
@@ -15,6 +15,7 @@ class NodeMutatedEvent(InfrahubBranchEvent):
     node_id: str = Field(..., description="The ID of the mutated node")
     action: MutationAction = Field(..., description="The action taken on the node")
     data: dict[str, Any] = Field(..., description="Data on modified object")
+    fields: list[str] = Field(default_factory=list, description="Fields provided in tha mutation")
```

Comment:
`tha` might be a little too casual

Reply:
In what way? That it needs another description or that we should do something completely different? The idea is to capture the name of attributes or relationships that have been modified.

Reply:
it was a joke. I assume `tha` is a misspelling of `the`

Reply:
😘

Reply:
Will fix that.

Reply:
I've corrected this typo. I didn't see the one I made when looking at your comment in the mobile client so I didn't quite understand what you meant :)